app/models/notifications/mentioned.rb

```diff
@@ -1,38 +1,29 @@
 module Notifications
-  class Mentioned < Notification
+  module Mentioned
+    extend ActiveSupport::Concern
+
     def mail_job
       Workers::Mail::Mentioned
     end
 
-    def popup_translation_key
-      "notifications.mentioned"
-    end
-
-    def deleted_translation_key
-      "notifications.mentioned_deleted"
-    end
-
     def linked_object
       target.mentions_container
     end
 
-    def self.notify(mentionable, recipient_user_ids)
-      actor = mentionable.author
-      relevant_mentions = filter_mentions(
-        mentionable.find_or_create_mentions.local.where.not(person: mentionable.author),
-        mentionable,
-        recipient_user_ids
-      )
+    included do
+      def self.notify(mentionable, recipient_user_ids)
```

Comment:
When using `AS::Concern`, instead of this, define a nested module called `ClassMethods` which have the class methods defined as instance methods.